docs/running_psalm/issues/RedundantPropertyInitializationCheck.md

```diff
@@ -1,6 +1,16 @@
 # RedundantPropertyInitializationCheck
 
-Emitted when checking `isset()` on a non-nullable property. This issue indicate a redundant check for projects that initialize their properties in constructor.
+Unitialized properties are statically hard to analyze. To prevent mistakes, Psalm will enforce that all properties should be initialized.
```

Comment:
```suggestion
Unitialized properties are hard to statically analyze. To prevent mistakes, Psalm will enforce that all properties should be initialized.
```